.env

```diff
@@ -0,0 +1,2 @@
+
+API_KEY = 197cf43e7d01659ab1e8742f02fcaaa4
```

Comment:
you posted your API key to github. It is because you did not put your `.env` file in the `.gitignore` file. 
in your `.gitignore` file you need 
`node_modules` 
`.env`